mne/minimum_norm/tests/test_inverse.py

```diff
@@ -299,97 +284,87 @@ def test_inverse_operator_channel_ordering():
-                                      depth=0.8)
+                                      depth=depth)
     fwd = convert_forward_solution(fwd, force_fixed=True, surf_ori=True)
     fwd = fwd['sol']['data']
     want = np.arange(fwd.shape[1])
-    for method, lower, upper in (('MNE', 83, 87),
-                                 ('dSPM', 96, 98),
-                                 ('sLORETA', 100, 100),
-                                 ('eLORETA', 100, 100)):
-        inv_op = apply_inverse(evoked, inv_fixed, lambda2, method).data
-        loc = np.abs(np.dot(inv_op, fwd))
-        # Compute the percentage of sources for which there is no localization
-        # bias:
-        perc = (want == np.argmax(loc, axis=0)).mean() * 100
-        assert lower <= perc <= upper, method
-
-    #
-    # Loose orientation
-    #
+    inv_op = apply_inverse(evoked, inv_fixed, lambda2, method).data
+    loc = np.abs(np.dot(inv_op, fwd))
+    # Compute the percentage of sources for which there is no localization
+    # bias:
+    perc = (want == np.argmax(loc, axis=0)).mean() * 100
+    assert lower <= perc <= upper, method
+
+
+@pytest.mark.parametrize('method, lower, upper', [
+    ('MNE', 25, 35),
+    ('dSPM', 25, 35),
+    ('sLORETA', 35, 40),
+    ('eLORETA', 40, 45)])
+def test_localization_bias_loose(bias_params, method, lower, upper):
+    """Test inverse localization bias for loose minimum-norm solvers."""
+    evoked, fwd_orig, noise_cov = bias_params
     fwd = fwd_orig.copy()
     inv_free = make_inverse_operator(evoked.info, fwd, noise_cov, loose=0.2,
                                      depth=0.8)
     fwd = fwd['sol']['data']
     want = np.arange(fwd.shape[1]) // 3
-    for method, lower, upper in (('MNE', 25, 35),
-                                 ('dSPM', 25, 35),
-                                 ('sLORETA', 35, 40),
-                                 ('eLORETA', 40, 45)):
-        inv_op = apply_inverse(evoked, inv_free, lambda2, method,
-                               pick_ori='vector').data
-        loc = np.linalg.norm(np.einsum('vos,sx->vxo', inv_op, fwd), axis=-1)
-        # Compute the percentage of sources for which there is no localization
-        # bias:
-        perc = (want == np.argmax(loc, axis=0)).mean() * 100
-        assert lower <= perc <= upper, method
-
-    #
-    # Free orientation
-    #
+    inv_op = apply_inverse(evoked, inv_free, lambda2, method,
+                           pick_ori='vector').data
+    loc = np.linalg.norm(np.einsum('vos,sx->vxo', inv_op, fwd), axis=-1)
+    # Compute the percentage of sources for which there is no localization
+    # bias:
+    perc = (want == np.argmax(loc, axis=0)).mean() * 100
+    assert lower <= perc <= upper, method
+
+
+@pytest.mark.parametrize('method, lower, upper, kwargs, depth', [
+    ('MNE', 35, 40, {}, dict(limit_depth_chs=False)),
+    ('MNE', 45, 55, {}, 0.8),
+    ('MNE', 65, 70, {}, dict(limit_depth_chs='whiten')),
```

Comment:
FYI this shows the performance improvement for MNE when using `whiten`-based depth weighting (at least in this case, with this measure for localization bias)